backend/app/models/auth.py

```diff
@@ -0,0 +1,27 @@
+from sqlmodel import Field, SQLModel
+
+
+# Generic message
+class Message(SQLModel):
+    message: str
+
+
+# JSON payload containing access token
+class Token(SQLModel):
+    access_token: str
+    token_type: str = "bearer"
+
+
+# Contents of JWT token
+class TokenPayload(SQLModel):
+    sub: str | None = None
+
+
+class NewPassword(SQLModel):
```

Comment:
I wonder if we should prioritize ripping all this functionality out. We're carrying this code from one place to another, bearing the cost, while we know it's all obsolete.

Reply:
That is a great point. Do you think I should do the work an include it in this PR?